attachment_synchronize/tests/test_import.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2020 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from .common import SyncCommon
+
+
+class TestImport(SyncCommon):
+    @property
+    def archived_files(self):
+        return self.backend._list(self.directory_archived)
+
+    @property
+    def input_files(self):
+        return self.backend._list(self.directory_input)
+
+    def _check_attachment_created(self, count=1):
+        attachment = self.env["attachment.queue"].search([("name", "=", "bar.txt")])
+        self.assertEqual(len(attachment), count)
+
+    def test_import_with_rename(self):
+        self.task.write({"after_import": "rename", "new_name": "foo.txt"})
```

Comment:
Could you test  rename with jinja pattern ?

Reply:
kesako ?

Reply:
I think is not necessary because def _template_render method is a basic code.